graphs/DataVisualisation/index.js

```diff
@@ -66,21 +75,26 @@ module.exports = React.createClass({
   },
 
   renderDeltaArrow: function() {
-    var props   = this.props,
-        delta   = props.delta,
-        loading = props.loading;
+    var props      = this.props,
+        delta      = props.delta,
+        loading    = props.loading,
+        emptyState = props.emptyState;
 
-    if (loading || delta) {
+    if (emptyState) {
+      return <DeltaArrow delta={ delta } emptyState={ emptyState } />;
+    }
+    else if (loading || delta) {
```

Comment:
You don't need the else if here. The return in the above statement mean you will never reach this point if the first if is true.